pkg/labels/labels.go

```diff
@@ -542,10 +556,12 @@ func (l Labels) IsReserved() bool {
 }
 
 // Has returns true if l contains the given label.
-func (l Labels) Has(label Label) bool {
-	for _, lbl := range l {
-		if lbl.matches(&label) {
-			return true
+func (l Labels) Has(args ...Label) bool {
+	for _, label := range args {
+		for _, lbl := range l {
+			if lbl.matches(&label) {
+				return true
+			}
```

Comment:
I don't understand this change and why it's necessary. Can you clarify it? Thank you

Reply:
It gets used [here](https://github.com/cilium/cilium/pull/22625/files#diff-bb1165de1a65911d59ac7f9a002aa9f1e25c30e073547adec11b174a174fdca6R431).

Reply:
I suppose I could just or together the `Has` invocations.